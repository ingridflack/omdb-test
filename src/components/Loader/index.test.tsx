import { shallow } from "enzyme";
import Loader from ".";

describe("Loader", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
  });
});
