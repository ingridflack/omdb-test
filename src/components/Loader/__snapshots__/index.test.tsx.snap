// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match snapshot 1`] = `
<styled.div>
  <Styled(AiOutlineLoading3Quarters) />
</styled.div>
`;
